workflow-service/src/main/java/com/redhat/parodos/workflow/execution/dto/WorkFlowStatusResponseDTO.java

```diff
@@ -45,6 +46,6 @@ public class WorkFlowStatusResponseDTO {
 	private WorkStatus status;
 
 	@JsonInclude(JsonInclude.Include.NON_EMPTY)
-	private List<WorkStatusResponseDTO> works;
+	private transient List<WorkStatusResponseDTO> works;
```

Comment:
if this one is `transient`, shouldn't the annotation be changed to `@JsonIgnore` or should it remain serialized to json?

Reply:
we need it to be serialized.  sonarlint requires jsonignore or transient, so I have to pick one

Reply:
this is dto, so I think transient won't afffect

Reply:
i rolled back all the serialzable implement.   obj.hashcode() doesn't need it